app/views/me/notices/index.html.haml

```diff
@@ -1,12 +1,24 @@
-.h2.first=:notices.t
+%div
+  %span.h2.first=:notices.t
+  - if @notices.total_pages < 2 && @notices.any?
+    %span.h2.dissmiss-all-notices-button.dismiss-all-wrapper.pull-right
```

Comment:
It is special class to handle inline right-aligned position for the button in case when there is not pagination block.
Without it button displays near the `Notices` title or crosses the notices list table.